MarkAndJimsDealOfTheDay/PlacingOrders/PotentialOrder.cs

```diff
@@ -1,46 +1,10 @@
-using System;
+    using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc;
+using MarkAndJimsDealOfTheDay.PlacingOrders;
 
 namespace MarkAndJimsDealOfTheDay.Controllers
 {
-    [Produces("application/json")]
-    [Route("api/PlaceOrder")]
-    public class PlaceOrderController : Controller
-    {
-        public IActionResult Post(PlaceOrderRequest request)
-        {
-            if (!PotentialOrder.CanBePlaced(request.customerId, request.productCode, request.quantity))
-            {
-                return BadRequest(PotentialOrder.GetPlacingErrors(request.customerId, request.productCode, request.quantity));
-            }
-
-            return Ok();
-        }
-    }
-
-    public class PlaceOrderService
-    {
-        private readonly IUow _uow;
-
-        public PlaceOrderService(IUow uow)
-        {
-            _uow = uow;
-        }
-
-        public void PlaceOrder(Guid customerId, string productCode, int quantity)
-        {
-            var result = PotentialOrder.BuildOrder(customerId, productCode, quantity);
-
-            _uow.Repository.Save(result.Entity);
-            _uow.Bus.Publish(result.Event);
-            _uow.Commit();
-        }
-    }
-
     public class PotentialOrder : IEntity
     {
```

Comment:
Probably should have an orderId:

public Guid OrderId {get;}

then constructor is:
private PotentialOrder(Guid orderId, Guid customerId, string productCode, int quantity)
... etc